source/frontend/src/mocks/featureset.mock.ts

```diff
@@ -1,33 +1,38 @@
-import { LocationFeatureDataset } from '@/components/common/mapFSM/useLocationFeatureLoader';
+import {
+  LocationFeatureDataset,
+  SelectedFeatureDataset,
+} from '@/components/common/mapFSM/useLocationFeatureLoader';
 
 import getMockISSResult from './mockISSResult';
 
-export const getMockLocationFeatureDataset = (): LocationFeatureDataset =>
-  ({
-    location: {
-      lat: 48.432802005,
-      lng: -123.310041775,
-    },
-    fileLocation: {
-      lat: 48.432802005,
-      lng: -123.310041775,
-    },
-    pimsFeature: {
+export const getMockLocationFeatureDataset = (): LocationFeatureDataset => ({
+  location: {
+    lat: 48.432802005,
+    lng: -123.310041775,
+  },
+  fileLocation: {
+    lat: 48.432802005,
+    lng: -123.310041775,
+  },
+  pimsFeatures: [
+    {
       properties: null,
       type: 'Feature',
       geometry: {
         type: 'Point',
         coordinates: [-123.310041775, 48.432802005],
       },
     },
-    parcelFeature: {
+  ],
+  parcelFeatures: [
+    {
       type: 'Feature',
       id: 'WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW.fid--55c55c24_18d9ff61aac_-5aaa',
       geometry: {
         type: 'Polygon',
         coordinates: [[[-123.31014591, 48.43274258]]],
       },
-      geometry_name: 'SHAPE',
+      //geometry_name: 'SHAPE',
```

Comment:
?

Reply:
parcel types do not have a geometry_name on the type coming from the geojson library. We could extend it and add it given that our instance does come with it.